### DISCUSSION TITLE
Welcome to learnr Discussions!

### DISCUSSION THREAD

**Discussion by gadenbuie** [rstudio/learnr#617] [Announcements]
## 👋 Welcome!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us how you use `learnr`! If you've written a learnr tutorial or have developed a package of learnr tutorials, we'd love to hear about it. Please include a link to your tutorial or package!

**Comment by rpruim** [comment]
Thanks for getting the discussion going.

I've written many `learnr` tutorials for classes, for workshops, and as an introduction to plotting in the `ggformula` package.  I tend to have bursts and lulls of `learnr` activity depending what else I'm doing.  I expect to have another burst in the next month as I prepare for an online course (a modeling-centric introduction to statistics) that begins in January.

I'm a long-time follower of the github repo, so I've seen traffic related to updates to `learnr`, but I haven't really been able to keep track of everything.  I'd love to see a post about the most interesting new things in `learnr` over the last 12-18 months so I can take advantage of what wasn't there the last time I had a `learnr` burst.

I'll end with a couple questions: 
* When do you plan to have another CRAN release?
* What is that current state of things like `gradethis` and `checkr` or whatever else is out there to score "quizzes" embedded in `learnr` tutorials?

**Reply by gadenbuie** [reply]
Hi @rpruim! Thanks for sharing – and please feel free to post links to your tutorials in the [Show and Tell](https://github.com/rstudio/learnr/discussions/categories/show-and-tell) category, we'd love to see them 😄 

> When do you plan to have another CRAN release?

We're much closer to a CRAN release than we've been in a while... I'd really like to send learnr to CRAN by the end of the year. 🤞 

I'm working on moving the package documentation from a custom R Markdown website to a new pkgdown site. Part of that work will be to organize the updates from the last 12-18 months into a coherent NEWS file, which will lay the foundation for a blog post recap of those changes. Ideally we'll be able to also send `gradethis` and a few other grading packages to CRAN so we can include those in the recap as well. That post will likely go up on the [RStudio blog](https://www.rstudio.com/blog/) and I'll be sure to cross post it here.

**Reply by gadenbuie** [reply]
Turns out I meant to say "by the end of next year". But learnr v0.11.0 is now on CRAN! Read all about the changes in [the learnr v0.11.0 release notes](https://pkgs.rstudio.com/learnr/articles/releases/learnr_v0-11-0.html).

**Comment by ninahauserberlin** [comment]
Hello everyone (and to @gadenbuie especdially),

I submitted the German translations but they linger around as a pull request. Could it be integrated soon? We would love to introduce other civic society organizations to learnr but they would need a German version.

Thanks a lot!

Best from Germany,
Nina

**Reply by gadenbuie** [reply]
This was on my task list for today! Thanks for the reminder 😄 I added a small comment in the first PR about enabling a setting that will allow me to finish up adding those translations to the package.

**Reply by ninahauserberlin** [reply]
Jippie! Im glad - thanks a lot! :) I think that is activated already - at least the checkbox is clicked in both. Maybe I need to change something else, too? 🤔

**Reply by ninahauserberlin** [reply]
![image](https://user-images.githubusercontent.com/66951297/145229453-927ff482-a345-4215-a484-13f399b202c3.png)
![image](https://user-images.githubusercontent.com/66951297/145229487-ab43e670-9fb8-49e0-a58c-e4016a315bb5.png)

**Reply by gadenbuie** [reply]
Oh I see. Maybe it's because your fork is actually a fork of a fork of learnr (a fork of `CorrelAid/learnr` instead of `rstudio/learnr`)? Anyway, no worries, I'll manage the merge manually. Thanks again!

**Reply by ninahauserberlin** [reply]
Oh no, how did this happen? Well thats what we get for forking and editing in our repo... Thanks a lot Garrick for taking the lead on this :) 
It is super cool that you are such a responsive and engaged team - extend my wishes to the team, please!

**Reply by gadenbuie** [reply]
No problem and thanks again for the translation! I just merged it in https://github.com/rstudio/learnr/pull/630
